solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -48,66 +43,96 @@ os_version          = "CentOS 7.9.2009 x86_64"
 region              = "ap-south-1"
 security_group_cidr = "134.204.222.36/32"
 key_name            = "devops-key"
-
+instance_count      = "3"
+ebs_volume_count    = "9"
+ebs_volume_size     = "10"
+tag_name            = "cortx-multinode"
 ```
 
-## Create AWS instance
-
-- Execute Terraform code (as shown below) to create AWS instance for CORTX Build and Deployment.  
-- The command will display public-ip on completion. Use this public-ip to connect AWS instance using SSH Protocol. 
+### Execute Instructions from Local Host to create AWS Instances and Network and Storage Configuration
+- Execute Terraform code (as shown below) to create AWS instances for CORTX Build and Deployment.
 ```
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
+- Execute following commands which will perform on all the nodes in the cluster,
+  - Setup network and storage devices for CORTX.
+  - Prompt for generating the `root` user password
+  - `/home/centos/setup.sh` will reboot all the nodes once executed
 
-## Network and Storage Configuration.
+**Note:**
+*The root password is required as a part of CORTX deployment.*
+```
+PUBLIC_IP=`terraform show -json terraform.tfstate | jq .values.outputs.aws_instance_public_ip_addr.value 2>&1 | tee ip.txt  | tr -d '",[]' | sed '/^$/d'`
+for ip in $PUBLIC_IP;do
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+done
+```
+- AWS instances are ready for CORTX Build and deployment now. Connect to EC2 nodes over SSH and validate that all three network cards has IP address assigned.
 
-- Execute `/home/centos/setup.sh` to setup Network and Storage devices for CORTX. Script will reboot instance on completion. 
+### CORTX Build
 
+- We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack.  
+- Become the root user by running the following command,
 ```
-ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance public-ip>" sudo bash /home/centos/setup.sh
+sudo su -
 ```
-- AWS instance is ready for CORTX Build and deployment now. Connect to instance over SSH and validate that all three network cards has IP address assigned.
-   
-- Generate `root` user password.  
-*The root password is required as a part of CORTX deployment.*
-   
+- Login to all the nodes over SSH from local host using public IP address and clone the cortx-re repository
```

Comment:
We will perform CORTX build on only one EC2 instance and no need to close cortx-re repo on all the nodes.